test/transaction-runner.ts

```diff
@@ -75,17 +78,17 @@ describe('TransactionRunner', () => {
   afterEach(() => sandbox.restore());
 
   describe('Runner', () => {
-    // tslint:disable-next-line no-any variable-name
-    let ExtendedRunner;
+    // tslint:disable-next-line variable-name
+    let ExtendedRunner: Any;
 
-    let runFn;
-    let runner;
+    let runFn: sinon.SinonStub<Any, Any>;
+    let runner: Runner<Any>;
```

Comment:
let runFn:sinon.SinonStub;
    let runner:trTypes.Runner<void>;

Reply:
Here Runner is Type of Runner<T>